gnomad_qc/v3/sample_qc/v2_pc_relate.py

```diff
@@ -0,0 +1,58 @@
+from gnomad_qc.v2.resources import qc_mt_path, get_gnomad_meta
+from gnomad_qc.v3.resources import gnomad_v2_qc_sites, v3_qc, meta, v2_v3_pc_relate_pca_scores, v2_v3_relatedness
```

Comment:
are these imports still accurate? should this be `from gnomad_qc.v3.resources.sample_qc import ...`?

Reply:
These come  from different submodules, so yes I  think they're good as-is.

Reply:
ah I didn't realize they were imported in the __init__. thanks!